docs/source/howto/index.rst

```diff
@@ -5,7 +5,10 @@ How-To Guides
 .. toctree::
    :maxdepth: 1
 
-   codes
+   setup_computers
+   ssh
+   setup_codes
+   run_codes
```

Comment:
Do we really want to have that many new top-level how-to sections?

Reply:
I would think that we could move some of those into a "Run on remote machines / supercomputers" top-level section or so.

Reply:
I agree here. At the very least `setup_computers` and `ssh` should go together and `setup_codes` could just be the first subheading of `run_codes` which used to be `codes` (so that name could actually be restored)

Reply:
closing as changed after leo's merges